fendermint/app/src/cmd/key.rs

```diff
@@ -123,6 +137,17 @@ pub fn read_public_key(public_key: &PathBuf) -> anyhow::Result<PublicKey> {
     Ok(pk)
 }
 
+pub fn read_private_key(private_key: &PathBuf) -> anyhow::Result<SecretKey> {
```

Comment:
```suggestion
pub fn read_secret_key_hex(private_key: &PathBuf) -> anyhow::Result<SecretKey> {
```

We have `read_secret_key` already and this does the same thing but with hex format. `private` and `secret` are synonyms and because there is no docs to say what it does its' confusing.